deployment/hasura/metadata/actions.graphql

```diff
@@ -144,14 +144,36 @@ type Query {
   getSequenceSeqJson(
     seqId: String!
     simulationDatasetId: Int!
-  ): GetSeqJsonResponse!
+  ): GetSeqJsonResponse! @deprecated(reason: "Use bulk query getSequenceSeqJsonBulk instead")
```

Comment:
These `@deprecated` fields seem to be ignored by Hasura right now :(
Good to have for documentation purposes tho.

Reply:
womp womp 😢